docs/operating-scylla/scylla-yaml.inc

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
```

Comment:
@annastuchlik is this markup for something? why does it show up?

Reply:
@avikivity Use the `:orphan:` directive to let a document be built, but notify Sphinx that it is not reachable via a toctree. It's mostly used for pages that are included and not added to any index.
However, the directive doesn't seem to be working (or necessary) in .inc files, so it should be removed from scylla-yaml.inc.

I've reviewed this PR, and the documentation builds without any errors or warnings. I think it can be safely merged.